vars/hadolintExecute.groovy

```diff
@@ -0,0 +1,95 @@
+import static com.sap.piper.Prerequisites.checkScript
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.Utils
+import groovy.transform.Field
+
+@Field def STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = [
+    /**
+     * Name of the docker image that should be used, in which node should be installed and configured. Default value is 'hadolint/hadolint:latest-debian'.
+     */
+    'dockerImage'
+]
+@Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus([
+    /**
+     * Name of the configuration file used locally within the step.
+     */
+    'configurationFile',
+    /**
+     * URL pointing to the .hadolint.yaml exclude configuration to be used for linting
+     */
+    'configurationUrl',
```

Comment:
maybe also explain that a yaml in the repo will prevent downloading from server ...